spec/features/admin/stores/admin_can_approve_store_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.feature "Admin Stores" do
+  before(:each) do
+    admin = create(:admin)
+    create(:store,name:"store_1", status: "active")
+    create(:store, name:"store_2", status: "pending")
+    create(:store, name: "store_3", status: "suspended")
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+  end
+
```

Comment:
The test setup in both of these tests is the same, we could combine them into one test file